bucket/helm-chart-testing.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.7.1",
+    "description": "Tool for testing Helm charts. It is meant to be used for linting and testing pull requests. It automatically detects charts changed against the target branch.",
+    "homepage": "https://github.com/helm/chart-testing/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/helm/chart-testing/releases/download/v3.7.1/chart-testing_3.7.1_windows_amd64.zip",
+            "hash": "218f6672e2145fc270e166c132c46db36005fb990d07df01635958d9daf99ab8"
+        },
+        "arm64": {
+            "url": "https://github.com/helm/chart-testing/releases/download/v3.7.1/chart-testing_3.7.1_windows_arm64.zip",
+            "hash": "f26d5ed42e1586c6553abdab0f369981169b363906125078edc350292c6fee83"
+        }
+    },
+    "post_install": [
+        "New-Item -ItemType Directory -Path \"~/.ct\" -Force | Out-Null",
```

Comment:
Is this post_install necessary? We don't want to make changes, that aren't strictly required for the functioning of the app, outside of the installation folder.

Reply:
Yes, it's 100% required or else `ct lint` and `ct lint-and-install` will not work.

Reply:
According to the docs:

> Note that linting requires config file for [yamllint](https://github.com/adrienverge/yamllint) and [yamale](https://github.com/23andMe/Yamale). If not specified, these files are search in the current directory, the .ct directory in current directory, `$HOME/.ct`, and `/etc/ct`, in that order.

Reply:
I strongly disagree, these files are primarily tool scoped and the other installation methods make sure they exist. Without these files the tool doesn't work without significant user interaction.